### ISSUE TITLE
comparison functions

### ISSUE THREAD

**Issue by michalfapso** [p12tic/libsimdpp#22]
Hi Povilas,

I have few issues with comparison functions:

``` cpp
int32x4 i = make_int(13);
int32x4 j = make_int(10);
mask_int32x4 m;
m = cmp_le(i, j); // compilation FAILS
m = cmp_le(10, i); // compilation FAILS
m = cmp_le(i, 10); // compilation FAILS

float32x4 i = make_float(10);
float32x4 j = make_float(13);
mask_float32x4 m;
m = cmp_le(i, j); // OK
m = cmp_le(10, j); // OK
m = cmp_le(j, 10); // OK
m = cmp_le(make_float(13.0f), make_float(10.0f)); // compilation FAILS
```

In the simdpp/core/cmp_le.h it seems that cmp_le takes only float arguments. What is then the preffered way to compare integers? This issue is probably present in all comparison functions.

Thanks for any hints,
Michal

P.S.: the division operator is missing in simdpp/simd.h. I just added

``` cpp
#include <simdpp/operators/f_div.h>
```

in my local copy to be able to use it.

**Comment by p12tic** [comment]
This will be fixed in 2.1.

**Comment by p12tic** [comment]
Fixed in f9944abb347f352269eac2082c4783ea683294bb.
